src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,26 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String author;
+    @Column(unique = true)
```

Comment:
Specify `nullable` parameter too (and in all other not null fields, so hibernate can correctly validate you schema)